lib/linguist/languages.yml

```diff
@@ -1182,6 +1182,15 @@ GAS:
   tm_scope: source.assembly
   ace_mode: assembly_x86
 
+GCC machine description:
```

Comment:
Close call. Every OCD coder like myself would've gone nuts over the discrepancy.